archetypes/carbon-component-archetype/src/test/resources/projects/component/reference/src/main/java/org/test/component/GreeterImpl.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Archetype tests generates a project and compares it against this reference project we have provided. The project archetype creates has the year 2015, therefore the reference project also needs to have the same.